### PR TITLE
Updated README.adoc

### PR DESCRIPTION
**Pull request by joepetrowski**
Added a new section with instructions of how to join the Krumme Lanke testnet.
Closes Issue #1217

### CODE REVIEW

**Comment by joepetrowski** [comment]
[clabot:check]

**Comment by gavofyork** [comment]
why no BBQ Birch instructions?

**Comment by joepetrowski** [comment]
Marking on ice due to new testnet #1280 coming soon

File: README.adoc [hunk 1]
```diff
@@ -50,7 +50,7 @@ Other examples include the parachain-heads extrinsic in Polkadot and the "note-m
...
 
-The runtime is API entry points are expected to be in the runtime's `api` module. There is a specific ABI based upon the Substrate Simple Codec (`codec`), which is used to encode and decode the arguments for these functions and specify where and how they should be passed. A special macro is provided called `impl_stubs`, which prepares all functionality for marshalling arguments and basically just allows you to write the functions as you would normally (except for the fact that there must be example one argument - tuples are allowed to workaround).
+The runtime's API entry points are expected to be in the runtime's `api` module. There is a specific ABI based upon the Substrate Simple Codec (`codec`), which is used to encode and decode the arguments for these functions and specify where and how they should be passed. A special macro is provided called `impl_stubs`, which prepares all functionality for marshalling arguments and basically just allows you to write the functions as you would normally (except for the fact that there must be one argument - tuples are allowed to work around).
```
Comment (bkchr) [inline README.adoc]: That is not correct anymore. The runtime api stuff in the runtime is generated by the `impl_runtime_apis!` macro.
The macro expects you to implement all the runtime api traits you require and then takes care to generate all the interfaces for you.

**Comment by gavofyork** [comment]
conflicts. will try to get in once resolved.

**Pull request merged**